ccmlib/node.py

```diff
@@ -1187,11 +1191,11 @@ def get_sstables(self, keyspace, column_family, ignore_unsealed=True, cleanup_un
                 files.remove(f)
             if (ignore_unsealed or cleanup_unsealed) and os.path.exists(f.replace('Data.db', 'TOC.txt.tmp')):
                 if cleanup_unsealed:
-                    print_("get_sstables: Cleaning up unsealed SSTable: {}".format(f))
+                    logger.debug("get_sstables: Cleaning up unsealed SSTable: %s", f)
```

Comment:
A general question for the while patch, not just this like:
You changed `{}` to `%s`. Is this necessary? Isn't `{}` supported by Python's logger as well?
That being said, I don't know what is needed to do (if at all possible) to switch logging style.

Reply:
I create a uniform format for all log messages, and that's the correct way (I think the format will be fine too)

I got an error for this log message
![image](https://user-images.githubusercontent.com/11705430/147567380-02d92180-6089-4697-9a20-d5f841428aa7.png)

when I use this way the software will still run
![image](https://user-images.githubusercontent.com/11705430/147567414-8e660151-d49a-4669-8088-5ff66d277c4f.png)

I got an error in the console
![image](https://user-images.githubusercontent.com/11705430/147567498-5075a709-f281-4939-ad15-def7ed34078b.png)


more details yon can found in this link
https://medium.com/swlh/why-it-matters-how-you-log-in-python-1a1085851205